register-form.php

```diff
@@ -0,0 +1,210 @@
+<?php
+ require_once('template-includes.php');
+
+/**
+ * This is the content from register_form.php
+ */
+
+if(isset($_POST['reg_user'])){
+	// initializing variables
+	
+	$first_name = mysqli_real_escape_string($db_connect , $_POST['first_name']);
+	$last_name = mysqli_real_escape_string($db_connect , $_POST['last_name']);
+	$email_address = mysqli_real_escape_string($db_connect , $_POST['email_address']);
+	$password = mysqli_real_escape_string($db_connect , $_POST['password']);
+	$password_confirmation = mysqli_real_escape_string($db_connect , $_POST['password_confirmation']);
+	$phone_number = mysqli_real_escape_string($db_connect , $_POST['phone_number']);
+	$company_name = mysqli_real_escape_string($db_connect , $_POST['company_name']);
+	$company_site = mysqli_real_escape_string($db_connect , $_POST['company_site']);
+	$company_description = mysqli_real_escape_string($db_connect , $_POST['company_description']);
+	//$company_image = mysqli_real_escape_string($db_connect , $_FILES['company_image']);
+	
+	$errors = array();
+	
+	// form validation
+	
+	if(empty($first_name)) {array_push($errors, "First name is required");}
+	if(empty($last_name)) {array_push($errors, "Last name is required");}	
+	if(empty($email_address)) {array_push($errors, "Email is required");}
+	if(empty($password)) {array_push($errors, "Password is required");}
+	if($password !== $password_confirmation) {array_push($errors, "Passwords do not match");}
+	
+	// Validate if email is admin or not
+	$email_address = filter_var($email_address, FILTER_SANITIZE_EMAIL);
+	if (filter_var($email_address, FILTER_VALIDATE_EMAIL)) {
+	$email_pattern = '/@devrix.com/';
+	
+	  if (preg_match($email_pattern, $email_address)){
+		$is_admin = 1;
+	  }else{
+		$is_admin = 0;
+	  }
```

Comment:
Here's another good practice. You should only use `else` if you really need it. In this case, you can do something like this:
```suggestion
          $is_admin = 0;
	  if ( preg_match( $email_pattern, $email_address ) ) {
		$is_admin = 1;
	  }
```

or even
```suggestion
$is_admin = preg_match( $email_pattern, $email_address ) ? 1 : 0 
```

Reply:
Fixed